templates/terraform/custom_expand/compute_ssl_certificate_name_name_prefix.go.erb

```diff
@@ -0,0 +1,29 @@
+<% if false # the license inside this if block pertains to this file -%>
```

Comment:
Two things!  One: you can just use 

```
<%#
Copyright 2017 ...
...
#%>
```

I know all the existing code does it this way but it's making me bonkers.  :)

Two: this is really "name or name prefix", right?  Might be worth naming it "templates/terraform/custom_expand/name_or_name_prefix.go.erb" instead.

Reply:
Done